sefaz/accesskey/validation.go

```diff
@@ -18,7 +18,27 @@ func Check(accessKey AccessKey) error {
 	return nil
 }
 
-// validate execute all sub-routines necessary to perform a full accesskey validation
+// CheckNFF checks if an access key is valid also considering NFF rules. It returns an error with an specific code based on
+// the validation proble
+func CheckNFF(accessKey AccessKey) error {
```

Comment:
Shouldn't this validation extend the already-existing `Check` function? It seems strange to me that the user is forced to choose between `Check` and `CheckNFF`